frontend/public/components/catalog/catalog-page.tsx

```diff
@@ -104,16 +104,22 @@ export class CatalogListPage extends React.Component<CatalogListPageProps, Catal
       projectTemplateItems = this.normalizeTemplates(projectTemplateMetadata);
     }
 
-    return _.sortBy(
-      [
-        ...clusterServiceClassItems,
-        ...imageStreamItems,
-        ...templateItems,
-        ...extensionItems,
-        ...projectTemplateItems,
-      ],
-      'tileName',
-    );
+    const items = [
+      ...clusterServiceClassItems,
+      ...imageStreamItems,
+      ...templateItems,
+      ...extensionItems,
+      ...projectTemplateItems,
+    ];
+
+    const filteredItems = _.reject(items, [
+      'obj',
+      'metadata',
+      'annotations',
+      'apps.kubernetes.io/internal-object',
```

Comment:
We can not use the `apps.kubernetes.io` namespace without approval from upstream. I agree with Evan and Sam about changing this annotation

Reply:
Right, we are going for `operators.operatorframework.io` for now as per https://github.com/operator-framework/operator-lifecycle-manager/pull/1097